players/group5/player_map.py

```diff
@@ -94,11 +103,17 @@ def _get_map_coordinates(self, player_relative_coordinates: List[int]) -> List[i
 
     def get_start_pos(self) -> List[int]:
         return self._get_player_relative_coordinates(self._start_pos)
+    
+    def set_end_pos(self, end_pos: List[int]):
+        self._end_pos = end_pos
 
     def get_end_pos(self) -> List[int]:
         if self._end_pos[0] is None or self._end_pos[1] is None:
             return [None, None]
         return self._get_player_relative_coordinates(self._end_pos)
+    
+    def get_current_pos(self) -> List[int]:
+        return self._get_player_relative_coordinates(self.cur_pos)
```

Comment:
@Yoni-Livstone @Jinha-Park so you guys decided to use player relative coordinates for tomorrow?

Reply:
given
```
    def _get_player_relative_coordinates(self, map_coordinates: List[int]) -> List[int]:
        return [map_coordinates[0] - self.cur_pos[0], map_coordinates[1] - self.cur_pos[1]]

```

this always returns 0,0 for SimplePlayerMap. 

Given that it may not be 0, 0 later when we change the map system (e.g., when we implement the interface differently for say AdvancedPlayerMap), I think it's great that you guys added this to the interface 👍  thank you! 🌮